Adding functionality for Show Convictions/Incidents

LeSiiN opened the pull request:
https://www.youtube.com/watch?v=D-qE-puPzw0

LeSiiN committed:
app.js 

app.js

LeSiiN committed:
dashboard.html

dashboard.html

LeSiiN committed:
style.css

style.css

LeSiiN committed:
fixed wrong z-index

fixed wrong z-index

LeSiiN committed:
added view incident to known incidents

added view incident to known incidents

LeSiiN committed:
added missing hover for View Incident and View Profile

added missing hover for View Incident and View Profile

LeSiiN committed:
added missing make-waypoint hover

added missing make-waypoint hover

LeSiiN committed:
added incident id after ,,View Incident''

added incident id after ,,View Incident''

xFutte commented on ui/app.js:
```diff
@@ -1121,6 +1146,22 @@ $(document).ready(() => {
+        $(".incidents-known-container").fadeOut(250);
+        $(".close-all").css("filter", "none");
+      }
+
```
You could extract this code into a function and simply parse the selectors for making it a bit more clean. This would look as follows:

```js
function closeContainer(selector) {
   if (
        $(selector).css("display") != "none"
      ) {
        shouldClose = false;
        $(selector).fadeOut(250);
        $(".close-all").css("filter", "none");
      }
}
```
where you then call the method with the selectors where the current code is:
```js
closeContainer(".convictions-known-container");
closeContainer(".incidents-known-container");
```

Just a small comment - up to you if you want to implement it :)

LeSiiN replied:
done <3

xFutte commented on ui/style.css:
```diff
@@ -1563,6 +1563,40 @@ span.tag-input[contenteditable]:empty::before {
     object-fit: cover;
 }
 
+.convictions-known {
```
You can actually do `.convictions-known, .incidents-known` here and this will cover both elements. This way you avoid duplicate CSS and can delete the styling from line 1639 - 1644. The same goes for the other styling properties.

LeSiiN replied:
done <3

LeSiiN committed:
added closeContainer Function and used it on the new container

added closeContainer Function and used it on the new container

LeSiiN committed:
removed some duplicated code

removed some duplicated code

LeSiiN committed:
little duplicate bug fixed

little duplicate bug fixed

LeSiiN committed:
,,At least, from my point of view, the word "Known" should be removed. Is it possible to have unknown convictions?''

,,At least, from my point of view, the word "Known" should be removed. Is it possible to have unknown convictions?''

LeSiiN committed:
now searching for convictions/incidents but looking inside of it :P

now searching for convictions/incidents but looking inside of it :P
